elasticdl/python/data/parallel_table_reader.py

```diff
@@ -0,0 +1,111 @@
+from multiprocessing import Process, Queue
+
+from odps import ODPS
+
+
+class ParallelTableReader(object):
+    def __init__(
```

Comment:
I suggest to add more descriptions about the design on the Parallel Reader

Reply:
Done